src/components/slideshow/style.css

```diff
@@ -141,4 +141,6 @@
 
 .slider-zoom-container {
     margin: auto;
+    max-height: 100% !important;
+    max-width: 100% !important;
```

Comment:
I tried without `!important` - seems to work too.

Reply:
problem seems to be fixed for firefox now. Can you elaborate on safari issue